akka-docs/src/main/paradox/persistence-query.md

```diff
@@ -199,7 +199,7 @@ Java
 
 ## Performance and denormalization
 
-When building systems using @ref:[Event sourcing](typed/persistence.md#event-sourcing-concepts) and CQRS ([Command & Query Responsibility Segregation](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/jj554200(v=pandp.10)?redirectedfrom=MSDN)) techniques
+When building systems using @ref:[Event sourcing](typed/persistence.md#event-sourcing-concepts) and CQRS ([Command & Query Responsibility Segregation](https://docs.microsoft.com/en-us/previous-versions/msp-n-p/jj554200\(v=pandp.10\)?redirectedfrom=MSDN)) techniques
```

Comment:
I'm confused. How many of these links do we have? Isn't the right approach in https://github.com/akka/akka/pull/29576/files ?

Reply:
The change in #29576 doesn't seem to work as expected. See https://doc.akka.io/docs/akka/current/typed/persistence.html#event-sourcing-concepts

Reply:
but I think that the change in 29576 was merged after 2.6.9 (current)
it looks good in https://doc.akka.io/docs/akka/snapshot/typed/persistence.html#event-sourcing-concepts

@ignasi35 ?

Reply:
Ah ok. Fine by me, but than in that case better to align. Either use `\` or `%28`